.ci/bump-opbeans-ruby.yml

```diff
@@ -46,6 +46,16 @@ sources:
     transformers:
       - trimprefix: "v"
 
+conditions:
+  checkIfVersionDiffers:
+    name: Check if installed version differs
+    kind: shell
+    sourceid: elastic-apm-agent-ruby
+    spec:
+      command: bash .ci/scripts/test-version.sh
+      environments:
+        - name: PATH
```

Comment:
unrequired?

Reply:
The shell script won't be able to use `grep`, `head`, `awk` or any other command if the `PATH` is not added.


when I add the following lines to the script:

```
which grep
which head
which awk
which test
echo $version
```

this is the output without `PATH`:

```
checkIfVersionDiffers
---------------------
The shell 🐚 command "bash .ci/scripts/test-version.sh 4.6.2" exited on error (exit code 1) with the following output:
----
----

command stderr output was:
----
----
         => expected condition result false, got false
```


this is the output with `PATH`:

```
checkIfVersionDiffers
---------------------
The shell 🐚 command "bash .ci/scripts/test-version.sh 4.6.2" exited on error (exit code 1) with the following output:
----
/usr/bin/grep
/usr/bin/head
/usr/bin/awk
/bin/test
4.6.2
----

command stderr output was:
----
----
         => expected condition result false, got false
```

Reply:
Uhm, I'm a bit confused, I have not seen that behaviour when using `curl` in the sources section through a script:

- https://github.com/elastic/apm-pipeline-library/blob/bafc85a84dda5436498e1abd5ede9ee9499e6e6c/updatecli/updatecli.d/bump-aliases.yml#L67-L71


or a grep command in the target section:

- https://github.com/elastic/beats/blob/02d313c893e7daf78c6beb55df3c5c4ea0f0d338/.ci/bump-golang.yml#L79-L84

Reply:
In any case, I don't see this is a problem but the opposite, it might help when the environments are not configured

Reply:
That's interesting.

Due to the output not being generated when the `PATH` is not passes, I assumed it's not working.

But you are right, it seems to have the correct behaviour even without any output.

Reply:
https://github.com/elastic/opbeans-ruby/pull/186/commits/c1d7e595e3dfc5d51bfdde4aa6fccf80f888d965